bundles/io/org.eclipse.smarthome.io.transport.mdns/src/main/java/org/eclipse/smarthome/io/transport/mdns/ServiceDescription.java

```diff
@@ -44,4 +44,43 @@ public ServiceDescription(String serviceType, String serviceName, int servicePor
         this.serviceProperties = serviceProperties;
     }
 
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((serviceName == null) ? 0 : serviceName.hashCode());
+        result = prime * result + servicePort;
+        result = prime * result + ((serviceType == null) ? 0 : serviceType.hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
```

Comment:
I am pretty sure that your fomatter does not work correctly.
Also one liners should be arranged into curly brackets.

Reply:
That's auto-generated code from Eclipse, I never changed the templates.

Reply:
Hm, the ESH Eclipse IDE should use save actions that run the formatter.
The formatter should add the curly brackets.

I this does not work anymore then something has been lost.